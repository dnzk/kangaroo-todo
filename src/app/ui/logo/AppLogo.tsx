import kangarooTodo from "@public/kangaroo-todo.svg"
import Image from "next/image"

export default function AppLogo({ variant }: { variant: "large" | "medium" }) {
  return <Image
    src={kangarooTodo}
    alt="Kangaroo Todo"
    width={variant === "large" ? 181 : 113}
  />
}
