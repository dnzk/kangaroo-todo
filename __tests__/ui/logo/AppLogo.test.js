import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import AppLogo from '../../../src/app/ui/logo/AppLogo'

describe('AppLogo', () => {
  it('renders correctly', () => {
    const { container } = render(<AppLogo variant="large" />)
    expect(container).toMatchSnapshot()
  })
})
