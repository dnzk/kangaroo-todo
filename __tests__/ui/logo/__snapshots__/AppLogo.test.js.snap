// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogo renders correctly 1`] = `
<div>
  <img
    alt="Kangaroo Todo"
    data-nimg="1"
    decoding="async"
    height="181"
    loading="lazy"
    src="/_next/image?url=%2Fimg.jpg&w=384&q=75"
    srcset="/_next/image?url=%2Fimg.jpg&w=256&q=75 1x, /_next/image?url=%2Fimg.jpg&w=384&q=75 2x"
    style="color: transparent;"
    width="181"
  />
</div>
`;
